crates/paimon/src/spec/schema_change.rs

```diff
@@ -0,0 +1,736 @@
+    }
+
+    /// impl the `add_column_with_comment`.
+    pub fn add_column_with_comment(
+        field_name: String,
+        data_type: DataTypeRoot,
+        comment: String,
+    ) -> Self {
+        SchemaChange::AddColumn(AddColumn::new(field_name, data_type, Some(comment), None))
+    }
+
+    /// impl the `add_column_with_comment_and_move`.
+    pub fn add_column_with_comment_and_move(
+        field_name: String,
+        data_type: DataTypeRoot,
+        comment: String,
+        move_: Move,
+    ) -> Self {
+        SchemaChange::AddColumn(AddColumn::new(
+            field_name,
+            data_type,
+            Some(comment),
+            Some(move_),
+        ))
+    }
+
+    /// impl the `rename_column`.
+    pub fn rename_column(field_name: String, new_name: String) -> Self {
+        SchemaChange::RenameColumn(RenameColumn::new(field_name, new_name))
+    }
+
+    /// impl the `drop_column`.
+    pub fn drop_column(field_name: String) -> Self {
+        SchemaChange::DropColumn(DropColumn::new(field_name))
+    }
+
+    /// impl the `update_column_type`.
+    pub fn update_column_type(field_name: String, new_data_type: DataTypeRoot) -> Self {
+        SchemaChange::UpdateColumnType(UpdateColumnType::new(field_name, new_data_type))
+    }
+
+    /// impl the `update_column_position`.
+    pub fn update_column_nullability(field_name: String, new_nullability: bool) -> Self {
+        SchemaChange::UpdateColumnNullability(UpdateColumnNullability {
+            field_names: vec![field_name],
+            new_nullability,
+        })
+    }
+
+    /// impl the `update_columns_nullability`.
+    pub fn update_columns_nullability(field_names: Vec<String>, new_nullability: bool) -> Self {
+        SchemaChange::UpdateColumnNullability(UpdateColumnNullability {
+            field_names,
+            new_nullability,
+        })
+    }
+
+    /// impl the `update_column_comment`.
+    pub fn update_column_comment(field_name: String, comment: String) -> Self {
+        SchemaChange::UpdateColumnComment(UpdateColumnComment {
+            field_names: vec![field_name],
+            new_description: comment,
+        })
+    }
+
+    /// impl the `update_columns_comment`.
+    pub fn update_columns_comment(field_names: Vec<String>, comment: String) -> Self {
+        SchemaChange::UpdateColumnComment(UpdateColumnComment {
+            field_names,
+            new_description: comment,
+        })
+    }
+
+    /// impl the `update_column_position`.
+    pub fn update_column_position(move_: Move) -> Self {
+        SchemaChange::UpdateColumnPosition(UpdateColumnPosition::new(move_))
+    }
+}
+
+/// The type of move.
+///
+/// Reference: <https://github.com/apache/paimon/blob/release-0.8.2/paimon-core/src/main/java/org/apache/paimon/schema/SchemaChange.java#L412>
+#[derive(Debug, Clone, PartialEq, Eq, Deserialize, Serialize)]
+pub enum MoveType {
+    FIRST,
```

Comment:
The java version seems to have only:

```java
public enum MoveType {
    FIRST,
    AFTER
}
```

Reply:
> In the master dev, I have found the [MoveType](https://github.com/apache/paimon/blob/master/paimon-core/src/main/java/org/apache/paimon/schema/SchemaChange.java#L412) which contains `FIRST`, `AFTER`, `BEFORE` and `LAST`, so I added all in this enum.

Please refer to 0.8.2 now, we can add them later.

Reply:
> > In the master dev, I have found the [MoveType](https://github.com/apache/paimon/blob/master/paimon-core/src/main/java/org/apache/paimon/schema/SchemaChange.java#L412) which contains `FIRST`, `AFTER`, `BEFORE` and `LAST`, so I added all in this enum.
> 
> Please refer to 0.8.2 now, we can add them later.

OK, I will remove them.